From 5ae8209afad9a4284723712b46d8685e7f7fd72c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20=C5=81yszczek?= <michal.lyszczek@bofc.pl>
Date: Mon, 6 May 2019 00:06:39 +0200
Subject: [PATCH] sh/init.sh.Linux.in: change /run/lock from root:uucp to
 root:daemon
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

On gentoo /run/lock is owned by uucp group because of historical
reasons. However uucp does not exist on buildroot by default, and
it makes more sense that 'daemon' group should own this directory.

Signed-off-by: Michał Łyszczek <michal.lyszczek@bofc.pl>
[Fabrice: update for 0.43.3]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 sh/init.sh.Linux.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sh/init.sh.Linux.in b/sh/init.sh.Linux.in
index 222bbd3b..7f1a88db 100644
--- a/sh/init.sh.Linux.in
+++ b/sh/init.sh.Linux.in
@@ -85,7 +85,7 @@ fi
 fi
 
 checkpath -d "$RC_SVCDIR"
-checkpath -d -m 0775 -o root:uucp /run/lock
+checkpath -d -m 0775 -o root:daemon /run/lock
 
 # Try to mount xenfs as early as possible, otherwise rc_sys() will always
 # return RC_SYS_XENU and will think that we are in a domU while it's not.
-- 
2.18.1

